src/MIDebugEngine/Engine.Impl/ExceptionManager.cs

```diff
@@ -434,11 +434,26 @@ private async Task UpdateCatagory(Guid categoryId, ExceptionCategorySettings cat
             {
                 ExceptionBreakpointStates newCategoryState = updates.NewCategoryState.Value;
                 categorySettings.CategoryState = newCategoryState;
+
+                if (newCategoryState != ExceptionBreakpointStates.BreakThrown)
+                {
+                    ulong breakpointId;
+                    if (categorySettings.CurrentRules.TryGetValue("*", out breakpointId))
+                    {
+                        await _commandFactory.RemoveExceptionBreakpoint(categoryId, new ulong[] { breakpointId });
+                    }
+                    if (categorySettings.CurrentRules.TryGetValue("winrt::hresult_error", out breakpointId))
+                    {
+                        await _commandFactory.RemoveExceptionBreakpoint(categoryId, new ulong[] { breakpointId });
+                    }
```

Comment:
Sorry if this is just prototype code and it is obvious, but I believe you want: `await _commandFactory.RemoveExceptionBreakpoint(categoryId, categorySettings.CurrentRules.Values);`

Reply:
Yes, this is perfect. Thank you for bringing this up :)